cfpb/cfgov-refresh
# Regs3k: Add a notification block

This PR adds a notification block to the Regulations Full Width Text block that is used on the Regulations landing page. We have in the past frequently needed to be able to add notifications to eRegulations to inform users about deviations from current law and other distinctions between specific regulations in the tool and their official copies. 

This adds the ability to do that on both the main landing page with the listing of all regulations as well as the individual regulation landing pages. 

This is necessary for displaying our launch notification. 

## Testing

1. Visit the regulations landing page (http://localhost:8000/policy-compliance/rulemaking/regulations/) and edit it in the Wagtail Admin
2. Add a Notification to the Full Width Text block with your desired information.
3. Preview the page to see the notification.

## Screenshots

![image](https://user-images.githubusercontent.com/10562538/45378215-d8325a80-b5ca-11e8-97c4-285a144aa818.png)

![image](https://user-images.githubusercontent.com/10562538/45377948-1a0ed100-b5ca-11e8-9a41-fd9065dafefa.png)

## Todos

@contolini I think the styling looks good to me, but you might take a look over it to make sure what I've done makes sense.

## Checklist
